src/customer/structures/responses.ts

```diff
@@ -20,7 +20,7 @@ export interface ListThreadsResponse {
 export interface GetChatResponse {
   id: string;
   users: User[];
-  thread: Thread;
+  thread?: Thread;
```

Comment:
```suggestion
  thread: Thread;
```
let's not change anything in customer api now

Reply:
OK in v3.5